extension/src/testResourceManager.ts

```diff
@@ -1,18 +1,23 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-import { workspace, Uri } from 'vscode';
+import { commands, workspace, Uri } from 'vscode';
+import * as Commands from './commands';
+import { Logger } from './logger';
```

Comment:
Not use of Logger?

Reply:
e, i was implementing the feature for test explorer and found these bugs so i split the changes about the bug fixing to this pr